### DISCUSSION TITLE
Customising the output format of the Javascript plugin

### DISCUSSION THREAD

**Discussion by enzedonline** [danirus/django-comments-xtd#321] [Q&A] [answered]
Hello,

I've worked through the tutorial, set up my commenting with customised templates to make it consistent with my site.

I enabled the javascript plugin at the end (https://django-comments-xtd.readthedocs.io/en/latest/tutorial.html#javascript-plugin) but found it outputs in default format, none of the threads formatted, using the gravatar instead of my custom model etc..

I don't see where it pulls the formatting from. Does anyone have info on how to modify the output? 

Thanks in advance.

**Comment by danirus** [comment, marked as answer]
Hi @enzedonline, to get the design provided by bootstrap you need to add this entry in your template:

    <link href="{% static 'django_comments_xtd/css/bootstrap.min.css' %}" rel="stylesheet">

I forgot to mention it in the tutorial.

**Reply by enzedonline** [reply]
Hi @danirus  - sorry for slow response, I thought I'd said thanks already.
I've customised xtd quite heavily, it was the arrangement of components etc that I was looking to change around in the javascript output. I've stayed with HTML in the end.
